content/blog/sprint-review-2010.md

```diff
@@ -0,0 +1,41 @@
+Title: Wazo Platform 20.10 Released
+Date: 2020-07-20
+Author: The Wazo Authors
+Category: Wazo Platform
+Tags: wazo-platform, development
+Slug: release-review-2010
+Status: published
+
+Hello Wazo Platform community!
+
+Here is a short review of the Wazo Platform 20.10 release.
+
+## Technical features
+
+* **Asterisk**: Asterisk was upgraded to 17.5.1. See the [Asterisk release announcement](https://www.asterisk.org/downloads/asterisk-news/asterisk-1751-now-available)
+* **Performance**: We have finished removing the use of encryption for all internal HTTP communication between Wazo components. Encryption now only happens when Wazo is sending/receiving data from the outside world.
```

Comment:
We should remove this one:
* Already announced in 20.09
* See [my comment about upgrade note](https://github.com/wazo-platform/wazo-platform.org/pull/210/files#r457267942)

Reply:
Fixed. Thank you!